settings/settings.ui

```diff
@@ -1094,7 +1094,7 @@ Once you feel comfortable, click &lt;b&gt;next&lt;/b&gt; to learn more details!<
                             <property name="can_focus">False</property>
                             <property name="margin_top">10</property>
                             <property name="margin_bottom">10</property>
-                            <property name="label" translatable="yes" comments="The text of the big tutorial button. Using two lines looks quite good.">&lt;big&gt;&lt;b&gt;Open the example menu!&lt;/b&gt;&lt;/big&gt;</property>
+                            <property name="label" translatable="yes" comments="The text of the big tutorial button.">&lt;big&gt;&lt;b&gt;Open the example menu!&lt;/b&gt;&lt;/big&gt;</property>
```

Comment:
I think there was some design change, wasn't there? One single line does look good, too IMO

Reply:
Yeah, I just forgot to update the comment.